azavea/climate-change-lab
# Add missing clean:dist npm script

## Overview

This is the only _required_ change to get a working dist build. There are still issues where I can't seem to get files in `src/app/assets` to copy properly into the dist build. But that only affects display, not functionality. This unblocks #52.

## Testing Instructions

 * `npm run build:dist`
 * Start an http-server of your choice on a port of your choice, serving the files generated in the `./dist/` folder
 * The app should work with only the static image assets missing (which for now is only the svg image in the navbar
